map_processor:
  ros_parameters:
    raw_map_topic: "/raw_map_points"
    clustered_map_topic: "/clustered_map_points"
    timeout: 5
    min_z: 0.0
    max_z: 2.2
    min_cluster_size: 25
    min_samples: 3
    cluster_selection_epsilon: 0.8
    alpha: 1.0
